src/pages/organize/[orgId]/journeys/[journeyId]/closed.tsx

```diff
@@ -21,18 +21,15 @@ const scaffoldOptions = {
 export const getServerSideProps: GetServerSideProps = scaffold(async (ctx) => {
   const { orgId, journeyId } = ctx.params!;
 
-  await ctx.queryClient.prefetchQuery(
-    ['org', orgId],
-    getOrg(orgId as string, ctx.apiFetch)
-  );
-  const orgState = ctx.queryClient.getQueryState(['org', orgId]);
-
   const apiClient = new BackendApiClient(ctx.req.headers);
-  const journey = await apiClient.get(
+  const organization = await apiClient.get<ZetkinOrganization>(
+    `/api/orgs/${orgId}`
+  );
```

Comment:
This is actually not necessary. Trying to retrieve the journey will already fail if the organization is wrong or not accessible. I realized this when I was working on the tasks PR.